### PR TITLE
Feature/ID-81 Generic input interface

### PR DESCRIPTION
**Pull request by DanShepherd27**

### CODE REVIEW

File: src/AutomatedCar/App.xaml.cs
Comment (DanShepherd27) [inline src/AutomatedCar/App.xaml.cs:1]: Not important, I've only deleted some unnecessary usings.

File: src/AutomatedCar/SystemComponents/InputHandling/KeyboardHandler.cs
Comment (DanShepherd27) [inline src/AutomatedCar/SystemComponents/InputHandling/KeyboardHandler.cs:1]: Not important, I've only deleted some unnecessary usings.

File: src/AutomatedCar/SystemComponents/Packets/PowertrainPacket.cs
Comment (DanShepherd27) [inline src/AutomatedCar/SystemComponents/Packets/PowertrainPacket.cs:1]: Not important, I've only deleted some unnecessary usings.

File: src/AutomatedCar/SystemComponents/Powertrain/Powertrain.cs
Comment (DanShepherd27) [inline src/AutomatedCar/SystemComponents/Powertrain/Powertrain.cs:1]: Type casting should be fixed once input device priority handling has been implemented. It's ok for now.

**Review by skincso — APPROVED** [review]

**Pull request merged**